### PR TITLE
Add gotemplate.io

### PR DESCRIPTION
**Pull request by ricardbejarano**

### CODE REVIEW

File: showcase/gotemplate.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+name: gotemplate.io
+logo_url: https://gotemplate.io/images/logo.png
```
Comment (torch2424) [inline showcase/gotemplate.md:3]: It seems like I am getting a 403 on this image request? 😯

**Review by torch2424 — CHANGES_REQUESTED** [review]
Ahhh! This is such a cool project and great use of WASM! Thank you for opening this Pull request! 😄 🙏🏾 

So, I tried out the deoply preview, and it seems like the logo is giving a 403. So I'm guess their might be something up with cross domain requests? I left a comment on the line that should be fixed.

But after that, I'll go ahead and merge this! Thank you! 😄 🙏🏾 🎉

File: showcase/gotemplate.md (hunk 1 above)
Comment (ricardbejarano) [reply showcase/gotemplate.md]: I think it should work now.

**Comment by ricardbejarano** [comment]
There you go, mate!

New URL has all origins allowed.

PS: it'd be great to test `logo_url` CORS in CI, if you have a free afternoon.

**Comment by torch2424** [comment]
@ricardbejarano 

> There you go, mate! New URL has all origins allowed.

Thank you very much for the change! 😄 🙏🏾  However, I'm trying it out on the deploy preview, and I still seem to be getting a 403?

Would you happen to know what's happening here? 🤔 😯 

Screenshot 2022-06-19 222110

> PS: it'd be great to test logo_url CORS in CI, if you have a free afternoon.

I went ahead and opened #104 for whenever I get the chance! 😄 🎉

**Comment by ricardbejarano** [comment]
Alright, now it's fixed.

It wasn't CORS (fwiw, I've now disabled CORS and it still works).
It was Cloudflare doing its job (hotlink protection).

Adding a WAF rule to bypass hotlink protection when Referer is this site fixed it.

**Comment by ricardbejarano** [comment]
Yup! I've confirmed somehow else that it works.
Feel free to merge.

**Review by torch2424 — APPROVED** [review]
I also see that it is now working! Thank you very much! Approving and merging! 😄 🎉

**Pull request merged**